support/scripts/checkpatch.pl

```diff
@@ -2548,6 +2548,25 @@ sub process {
 			     "Patch subject line does not follow Unikraft scheme: '[Selector]/[Component]: [Short message]'\n" . $herecurr);
 		}
 
+# Check that the first line in the commit message is below
+# 70 characters to comply with Github UI. This helps
+# avoid commit message summaries placed at the beginning
+# to overflow into the second line when viewing through
+# the Github UI.
+		if ($in_header_lines &&
+		    $line =~ /^Subject:.*$/i) {
+			my $commit_summary_line0 = $1 if
+				$lines[$linenr - 1] =~ /^Subject: \[.*\] (.*)$/;
+			my $commit_summary_line1 = $lines[$linenr];
+			my $commit_summary = $commit_summary_line0 .
+					     $commit_summary_line1;
+			if (length($commit_summary) > 70) {
+				WARN("COMMIT_LOG_LONG_LINE",
```

Comment:
I think it would make sense to introduce a different switch for this. I'd call it `COMMIT_SUBJECT_LONG_LINE` since it is about the length of the subject line only.